### DISCUSSION TITLE
econo_news/2024/09/23/%EC%97%90%EC%BD%94%EB%85%B8-%EB%89%B4%EC%8A%A4-9%EC%9B%94%ED%98%B8

### DISCUSSION THREAD

**Discussion by giscus** [JNU-econovation/JNU-econovation.github.io#19] [Show and tell]
# econo_news/2024/09/23/%EC%97%90%EC%BD%94%EB%85%B8-%EB%89%B4%EC%8A%A4-9%EC%9B%94%ED%98%B8

에코노 뉴스 9월호

https://jnu-econovation.github.io/econo_news/2024/09/23/%EC%97%90%EC%BD%94%EB%85%B8-%EB%89%B4%EC%8A%A4-9%EC%9B%94%ED%98%B8.html

**Comment by capDoYeonLee** [comment]
에코노 파이팅!
